chia/_tests/util/build_network_protocol_files.py

```diff
@@ -293,9 +293,8 @@ def main() -> None:
     for name, function in name_to_function.items():
         path = tests_dir.joinpath(name)
         path.write_text(function())
-        # black seems to have trouble when run as a module so not using `python -m black`
         subprocess.run(
-            [scripts_path.joinpath("black"), os.fspath(path.relative_to(tests_dir))],
+            [scripts_path.joinpath("ruff"), "format", os.fspath(path.relative_to(tests_dir))],
```

Comment:
this is what the comment was suggesting i preferred.  or at least i'm guessing that was me.  if it works it is a bit less tied to os behavior and more to python behavior.

```suggestion
            [sys.executable, "-m", "ruff", "format", os.fspath(path.relative_to(tests_dir))],
```

Reply:
Addressed in an accidentally poorly named commit